src/Moq/Moq.csproj

```diff
@@ -39,6 +39,10 @@
 		<PackageReference Include="Castle.Core" Version="4.3.1" />
 		<PackageReference Include="IFluentInterface" Version="2.1.0" />
 		<PackageReference Include="System.Threading.Tasks.Extensions" Version="4.3.0" />
+		<PackageReference Include="TypeNameFormatter.Sources" Version="1.0.0">
+		  <PrivateAssets>all</PrivateAssets>
+		  <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+		</PackageReference>
```

Comment:
I think you could simplify this to just `<PackageReference Include="TypeNameFormatter.Sources" Version="1.0.0" PrivateAssets="All" />` (but I might be wrong).

Reply:
i didn't add this manually.  It was added by VS.